lib/IDE/CompletionLookup.cpp

```diff
@@ -1825,6 +1838,8 @@ void CompletionLookup::addMacroExpansion(const MacroDecl *MD,
   if (!MD->getResultInterfaceType()->isVoid()) {
     addTypeAnnotation(Builder, MD->getResultInterfaceType(),
                       MD->getGenericSignature());
+  } else {
+    Builder.addTypeAnnotation("Macro");
```

Comment:
Adding macro role can be beneficial? (e.g. `Declaration Macro`, `Peer Macro`) but how about multi-role macros?